benchmarks/README.md

```diff
@@ -0,0 +1,11 @@
+# CP2K Benchmarks
+
+This directory contains input files for CP2K's benchmarks.
+
+For documentation on CP2K's input files, please refer to the [Input Reference Manual](https://manual.cp2k.org/).
+
```

Comment:
Maybe also add a link to https://www.cp2k.org/performance where we collect measurements from different machines.

Reply:
Thanks, good point. Done in [de27e43](https://github.com/cp2k/cp2k/pull/592/commits/de27e43599b9e8cbc180093e7cfe6bf348c4b9ae).